DiplomaTests/DiplomaTests/Utility/TableHelper.cs

```diff
@@ -0,0 +1,73 @@
+﻿using DiplomaTests.Pages;
+using OpenQA.Selenium;
+
+namespace DiplomaTests.Utility
+{
+    class TableHelper : PageBase
```

Comment:
instead of creating helper for table, you may create table component, because table it's actually a web element